interfaces/ISO15118_charger.yaml

```diff
@@ -255,6 +255,44 @@ cmds:
       status:
         description: "True: Cable check is okay"
         type: boolean
+  PnC_supporting_Cert_Installation_Update:
+    description: >-
+      ServiceDiscoveryRes - The charger supports the certificate installation/update
+      service and has a connection to an SA for this purpose
+    arguments:
+      status:
+        description: "True: The service is supported"
+        type: boolean
+  PnC_Response_Installation:
```

Comment:
We should try to find a comprehensive naming for the async publish (request) and command (response) functionality for example:
PnC_Request_Certificate_Install <-> Set_PnC_Response_Certificate_Install
or even better imo (because it is not directly PnC related):
Install_Certificate_Request <-> Set_Install_Certificate_Response
Update_Certificate_Request <-> Set_Update_Certificate_Response
and 
PnC_Authorization_Request <-> Set_PnC_Authorization_Response

Reply:
I like that, done!